src/main/java/codeu/model/data/Conversation.java

```diff
@@ -28,6 +28,9 @@
   public final UUID owner;
   public final Instant creation;
   public final String title;
+  /** This Boolean states whether or not the Conversation is private or not (by default is set to false).
+   * Only true when between two users become pals and want to message each other. */
+  public Boolean privateConversation;
```

Comment:
- does this need to be public? You have a getter method, I think it's best private. Same with the others - can you make them all private?
- can you make it final? Make instance variable final whenever possible, unless they can be changed, which doesn't seem like the case here
- I would declare this boolean (primitive) rather than Boolean (boxed class). I note that everywhere you call the constructor you are passing true instead of Boolean.True anyway. Generally stick with primitives unless there is a reason to do otherwise

Reply:
that's a good point, thanks @tucosh! The other instance variables were public, so I just made privateConversation public as well, but it makes more sense to be private. & declared this boolean to be a primitive as well!